packages/openapi-generator/src/generator.ts

```diff
@@ -141,3 +169,12 @@ async function generateOpenApiService(
 function parseServiceName(filePath: string): string {
   return parse(filePath).name.replace(/-openapi$/, '');
 }
+
+/**
+ * Get the current SDK version from the package json.
+ * @returns The SDK version.
+ */
+function getSDKVersion(): string {
```

Comment:
if we enable `"resolveJsonModule": true` in the tsconfig we can just use `import {version} from './package.json'`

Reply:
as discussed this leads to other problems so we go for the read method.